README.md

```diff
@@ -119,85 +119,9 @@ These are intended to be used with the [ImageResizer for .NET](https://imageresi
 
 ---
 
-## Project structure
+### ⚙️ Include VS Code tasks?
```

Comment:
Seeing this again makes me realize how weird this is. We moved this question and logic to `@creuna/cli`, so this package knows nothing about a `tasks.json`. On the other hand, having this documentation in docs for `cli` also seems out of place.